tools/test_symlink.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
+
+import os
+
+print "node_modules", os.path.islink("node_modules")
+print "core/libdeno/buildtools", os.path.islink("core/libdeno/buildtools")
```

Comment:
Handling symlinks isn't supported on Windows with Python 2.7 😞  (it was only added in Python 3)